specification/attestation/resource-manager/Microsoft.Attestation/stable/2020-10-01/attestation.json

```diff
@@ -807,15 +804,6 @@
           "$ref": "../../../../../common-types/rfcs/rfc7517.json#/definitions/JSONWebKeySet"
         }
       }
-    },
-    "PrivateEndpointConnectionProperties": {
```

Comment:
This object defnition is removed. Is it expected? Is it used by others?

Reply:
This was a duplicate definition of the type implemented in common-types/resource-manager/privatelink.json. Customers are expecting the definitions in privatelink.json, and that's what the service generates.

The private definition was included to work around a swagger breaking change flag, but now it is generating errors because of the duplicate definition.